satellite/console/consoleweb/server.go

```diff
@@ -90,6 +95,7 @@ func NewServer(logger *zap.Logger, config Config, service *console.Service, mail
 	if server.config.StaticDir != "" {
 		mux.Handle("/activation/", http.HandlerFunc(server.accountActivationHandler))
 		mux.Handle("/registrationToken/", http.HandlerFunc(server.createRegistrationTokenHandler))
+		mux.Handle("/usageReport/", http.HandlerFunc(server.bucketUsageReportHandler))
```

Comment:
I just noticed that we have upper-case and lowercase mixed in url paths, I suspect we should default to lowercase and with dashes.